create/application.go

```diff
@@ -45,6 +45,7 @@ type applicationCmd struct {
 	GitInformationServiceURL string            `help:"URL of the git information service." default:"https://git-info.deplo.io" env:"GIT_INFORMATION_SERVICE_URL" hidden:""`
 	SkipRepoAccessCheck      bool              `help:"Skip the git repository access check" default:"false"`
 	Debug                    bool              `help:"Enable debug messages" default:"false"`
+	Language                 string            `help:"Language specifies which language your app is. If left empty, deploio will detect the language automatically. Valid values are: ruby;php;python;golang;nodejs;static" default:"" hidden:""`
```

Comment:
Instead of just listing the valid values in the help you can use the `enum: ruby,php...` tag to automatically validate them

Reply:
added